src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -115,6 +117,15 @@ class KubeflowTrainingService implements TrainingService {
             const restServer: KubeflowJobRestServer = component.get(KubeflowJobRestServer);
             this.kubeflowRestServerPort = restServer.clusterRestServerPort;
         }
+        // initialize kubeflow trial config to specific type
+        let kubeflowTrialConfig;
+        if(this.kubeflowClusterConfig.operator === 'tf-operator') {
+            kubeflowTrialConfig = <KubeflowTrialConfigTensorflow>this.kubeflowTrialConfig;
+        }else if(this.kubeflowClusterConfig.operator === 'pytorch-operator'){
+            kubeflowTrialConfig = <KubeflowTrialConfigPytorch>this.kubeflowTrialConfig;
+        }else {
+            throw Error('kubeflowTrialConfig not initialized!')
```

Comment:
Suggest to change to:
throw Error(`operator ${this.kubeflowClusterConfig.operator} is invalid')

Reply:
update to throw Error(`operator ${this.kubeflowClusterConfig.operator} is invalid')